gnome-initial-setup/pages/account/account.css

```diff
@@ -0,0 +1,15 @@
+/* Colors from the Tango palette */
+levelbar trough block.filled.weak {
+    background-color: #ef2929;
+    border-color: #cc0000;
+}
+
+levelbar trough block.filled.regular {
+    background-color: #edd400;
+    border-color: #c4a000;
+}
+
+levelbar trough block.filled.good {
+    background-color: #73d216;
+    border-color: #4e9a06;
+}
```

Comment:
Pretty sure that at least some (warning/success) do... Perhaps not all three levels though. I am OK to have the custom theming here in any case.